### PR TITLE
Customize account modal, base of wallet and publisher component

### PR DESCRIPTION
**Pull request by KonradFak**

### CODE REVIEW

File: src/app/advertiser/advertiser.component.ts [hunk 1]
```diff
@@ -7,9 +13,39 @@ import { Component, OnInit, ViewEncapsulation } from '@angular/core';
...
+  handleChooseAccountDialog() {
+    const dialogRef = this.dialog.open(CustomizeAccountChooseDialogComponent);
+
+    dialogRef.afterClosed()
+      .subscribe(
+        (accounts) => this.hadleCustomizeDialog(accounts)
+      );
   }
 
+  hadleCustomizeDialog(accounts) {
```
Comment (BartoszBilejczyk) [inline src/app/advertiser/advertiser.component.ts]: typo - hadle -> handle

File: src/app/advertiser/advertiser.component.ts [hunk 2]
```diff
@@ -7,9 +13,39 @@ import { Component, OnInit, ViewEncapsulation } from '@angular/core';
...
+      setTimeout(() => this.handleChooseAccountDialog());
+    }
+  }
+
+  handleChooseAccountDialog() {
+    const dialogRef = this.dialog.open(CustomizeAccountChooseDialogComponent);
+
+    dialogRef.afterClosed()
+      .subscribe(
+        (accounts) => this.hadleCustomizeDialog(accounts)
```
Comment (BartoszBilejczyk) [inline src/app/advertiser/advertiser.component.ts]: typo - hadle -> handle

File: src/styles.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
```
Comment (BartoszBilejczyk) [inline src/styles.scss]: I know this is needed - we will probably need to make our own theme or make a 'fake' one, right? (sth like empty)

File: src/app/advertiser/advertiser.component.ts [hunk 4]
```diff
@@ -7,9 +13,39 @@ import { Component, OnInit, ViewEncapsulation } from '@angular/core';
...
+    private route: ActivatedRoute,
+    private dialog: MatDialog,
+    private router: Router
+  ) { }
 
   ngOnInit() {
+    const firstLogin = this.route.snapshot.queryParams['customize'];
+
+    if (firstLogin) {
+      // ExpressionChangedAfterItHasBeenCheckedError bug fix
```
Comment (aqum) [inline src/app/advertiser/advertiser.component.ts]: This error seems weird. Do you have additional details - which expression it affects?

**Pull request merged**